helmclient.go

```diff
@@ -116,68 +138,237 @@ func (c *Client) GetReleaseHistory(releaseName string) (*ReleaseHistory, error)
 		return nil, microerror.Maskf(tooManyResultsError, "%d releases found, expected 1", len(resp.Releases))
 	}
 
-	release := resp.Releases[0]
-	if release.Chart != nil && release.Chart.Metadata != nil {
-		version = release.Chart.Metadata.Version
-	}
+	var history *ReleaseHistory
+	{
+		release := resp.Releases[0]
 
-	history := &ReleaseHistory{
-		Name:    release.Name,
-		Version: version,
+		var version string
+		if release.Chart != nil && release.Chart.Metadata != nil {
+			version = release.Chart.Metadata.Version
+		}
+
+		history = &ReleaseHistory{
+			Name:    release.Name,
+			Version: version,
+		}
 	}
 
 	return history, nil
 }
 
 // InstallFromTarball installs a chart packaged in the given tarball.
 func (c *Client) InstallFromTarball(path, ns string, options ...helmclient.InstallOption) error {
-	_, err := c.helmClient.InstallRelease(path, ns, options...)
+	t, err := c.newTunnel()
+	if err != nil {
+		return microerror.Mask(err)
+	}
+	defer c.closeTunnel(t)
+
+	_, err = c.newHelmClientFromTunnel(t).InstallRelease(path, ns, options...)
 	if err != nil {
 		return microerror.Mask(err)
 	}
 
 	return nil
 }
 
+func (c *Client) InstallTiller() error {
+	var name = "tiller"
+	var namespace = "kube-system"
```

Comment:
We have `tillerDefaultNamespace`. Would also be nice to have a constant for the name.

Reply:
Good shout. Done.